Makefile

```diff
@@ -162,14 +167,14 @@ else
 endif
 
 convert-examples-to-%:
-	$(patsubst %, $(RUN) linkml-convert  % -s $(SOURCE_SCHEMA_PATH) -C Person, $(shell ${SHELL} find src/data/examples -name "*.yaml"))
+	$(patsubst %, $(RUN) linkml-convert  % -s $(SOURCE_SCHEMA_PATH) -C Entity, $(shell ${SHELL} find $(SOURCE_SAMPLE_DATA_DIR) -name "*.yaml"))
```

Comment:
we aren't actually using these shortcuts, but updating them just in case we do in the future.